en/docs/setup/deployment-guide.md

```diff
@@ -31,38 +31,13 @@ necessary system requirements and a compatible environment.
 </colgroup>
 <tbody>
 <tr class="odd">
-<td>Physical</td>
+<td></td>
```

Comment:
Why are we removing the section header?

Reply:
As we don't have multiple sections anymore.